packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -179,19 +179,23 @@ export async function pollDeploymentStatus(deployCtx: DeployContext) {
             currentStatus[operation.resourceName] = operation.status;
             if (!polledOperations.includes(operation.resourceName)) {
               polledOperations.push(operation.resourceName);
-
-              if (operation.subscriptionId === deployCtx.client.subscriptionId) {
-                const subOperations = await deployCtx.client.deploymentOperations.list(
-                  operation.resourceGroupName,
-                  operation.resourceName
-                );
-                subOperations.forEach((sub) => {
-                  const subOperation = getRequiredOperation(sub, deployCtx);
-                  if (subOperation) {
-                    currentStatus[subOperation.resourceName] = subOperation.status;
-                  }
-                });
+              let client = deployCtx.client;
+              if (operation.subscriptionId !== deployCtx.client.subscriptionId) {
+                const azureToken =
+                  await deployCtx.ctx.azureAccountProvider?.getAccountCredentialAsync();
```

Comment:
will there be any problem if we getAccountCredentialAsync before every pull status?

Reply:
Actually the getAccountCredentialAsync() is called to get the client of arm deployment before pull status. 
It will fail when preparing that client.